client/reader/stream/style.scss

```diff
@@ -661,6 +661,19 @@
 			overflow: hidden;
 			-webkit-line-clamp: 1;
 			-webkit-box-orient: vertical;
+			white-space: nowrap;
+			width: 225px;
```

Comment:
I'd recommend applying the width (or max-width?) only when the sidebar is rendered in a two column layout (use parent selectors `.stream__two-column .stream__right-column`). In the tabbed layout, the width of the "sidebar" can be the same as the main feed.

The max width of the sidebar in the two column layout is 270px;

Reply:
> I'd recommend applying the width (or max-width?) only when the sidebar is rendered in a two column layout (use parent selectors .stream__two-column .stream__right-column).

Updated in https://github.com/Automattic/wp-calypso/pull/78858/commits/8328cac1ca54ca08736e0fb726c017612e58697b

I believe 270px is the entire A tag. I need to isolate a width for just the text portion (minus the avatar and padding). 

![CleanShot 2023-06-30 at 15 28 09](https://github.com/Automattic/wp-calypso/assets/5634774/663862c5-4f6b-4156-a329-cd0dadbab460)

Settled on 225px since that seemed to work.